### PR TITLE
Implement file delete from disc

### PR DESCRIPTION
**Pull request by danielspofford**
Fulfills: #1

### CODE REVIEW

File: vialRackAPI/lib/storeFile.ex [hunk 1]
```diff
@@ -4,6 +4,8 @@ defmodule VialRackAPI.StoreFile do
 	Responsible for managing files (Uploading and deletion).
 	"""
 
+	@dir "priv"
```
Comment (danielspofford) [inline vialRackAPI/lib/storeFile.ex:7]: I think all of these "File is using tabs" comments are irrelevant to this PR, of the two files I edited I maintained consistency with what the file was already doing.

File: vialRackAPI/web/controllers/file_controller.ex [hunk 2]
```diff
@@ -73,22 +73,15 @@ defmodule VialRackAPI.FileController do
...
-				end
-
-        json conn_with_status(conn, :ok), :ok
-      _ ->
-        json conn |> put_status(:bad_request), %{errors: ["invalid request"]}
+  def delete(conn, %{"id" => id}) do
+    with {:get, file} when not is_nil(file) <- {:get, Repo.get(VialRackAPI.File, id)},
+         {:repo_delete, {:ok, _}} <- {:repo_delete, Repo.delete(file)},
+         {:disc_delete, :ok} <- {:disc_delete, VialRackAPI.StoreFile.delete(file)} do
+      json conn |> put_status(204), nil
```
[the remark this answers is not in the document]
Comment (danielspofford) [reply vialRackAPI/web/controllers/file_controller.ex:80]: The rest of the file uses one function long pipelines so I maintained consistency.

File: vialRackAPI/lib/storeFile.ex [hunk 3]
```diff
@@ -21,14 +23,33 @@ defmodule VialRackAPI.StoreFile do
...
+
+						Expected output  :ok
+						Error output     {:error, reason}
+
+	"""
+
+	def delete(%{file: path}) do
+		case File.rm("#{@dir}#{path}") do
+			:ok -> :ok
+			{:error, reason}-> {:error, reason}
```
Comment (danielspofford) [inline vialRackAPI/lib/storeFile.ex]: Will fix this, stemmed from a copy paste as this is an issue outside of my changes as well (will fix other instance within the same file as well).

**Comment by auyer** [comment]
Thanks for the help. Hope this project small can be helpful for some devs in the future.

**Pull request merged**